docs/administration-guide/10-single-sign-on.md

```diff
@@ -20,34 +18,47 @@ Now existing Metabase users signed into a Google account that matches their Meta
 
 ###  Enabling account creation with Google Sign-In
 
-If you’ve added your Google client ID to your Metabase settings you can also let users sign up on their own without creating accounts for them.
+If you’ve added your Google client ID to your Metabase settings, you can also let users sign up on their own without creating accounts for them.
 
 To enable this, go to the Google Sign-In configuration page, and specify the email domain you want to allow. For example, if you work at WidgetCo you could enter `widgetco.com` in the field to let anyone with a company email sign up on their own.
 
-Note: Metabase accounts created with Google Sign-In do not have passwords and must use Google to sign in to Metabase.
+Note that Metabase accounts created with Google Sign-In do not have passwords and must use Google to sign in to Metabase.
 
+## Enabling LDAP authentication
 
-### Enabling LDAP authentication
+If your organization uses LDAP, enabling it in Metabase will allow your users to log in via their LDAP credentials.
 
-If your organization uses LDAP, and you want to allow your users to log in via their LDAP credentials, you can do so as follows.
+To get started, click the **Configure** button in the LDAP section of the Admin Panel's Authentication page, where you'll fill out the form with the information about your LDAP server — hostname, port, security settings, and the LDAP admin username and password — and save your changes.
```

Comment:
Was trying to make the wording here vague enough that it could be applied to either the page’s current or future UI, but I’ll remove the backport tag so 42 onward only, yes?

Reply:
I think my only concern with backporting is that currently, it's an important step to click the toggle at the top of the form to enable LDAP prior to saving. Otherwise the LDAP settings will not be validated, and it won't actually be enabled. So we should keep that callout on line 38 in the docs until the 42 release.